src/task-builders/build/build-ts.js

```diff
@@ -22,49 +23,90 @@ module.exports = (project, options) => {
     const workingDir = rootDir.getChild('working');
 
     const dirs = ['src', 'test'];
-    if (project.projectType === 'aws-microservice') {
-        dirs.push('infra');
-    }
+    if(project.projectType ==! 'ui'){
```

Comment:
It looks like the file has not been formatted. You will need to run a format/lint/full test before submitting the PR.